### PR TITLE
PBS-2351 Sortie comments and remarks character limit

### PR DESCRIPTION
**Pull request by renicklopez**
### What this PR does :thinking: 
Add character limits to remarks and sortie comments

### Resolves :white_check_mark: 
[PBS-2351](https://jira.il2.dso.mil/browse/PBS-2351)

### CODE REVIEW

**Comment by tsanders88** [comment]
text area gets larger when attempting to type beyond the character limit for some reason

**Comment by renicklopez** [comment]
> doesn't display character limit
what do you mean? this is consistent with the rest of the app where there is a limit, we ust wont allow them to type after limit is reached

**Comment by Adam-Roney581** [comment]
> > text area gets larger when attempting to type beyond the character limit for some reason
> 
> I'm not getting this on my end, which text field remarks or sortie comments? which simulator or ipad size are you using?

I see Sortie Comments grows when reaching character limit, but 781 remarks scrolls. I was testing on an iPad Mini 6th Generation

**Comment by renicklopez** [comment]
> > > text area gets larger when attempting to type beyond the character limit for some reason
> > 
> > 
> > I'm not getting this on my end, which text field remarks or sortie comments? which simulator or ipad size are you using?
> 
> I see Sortie Comments grows when reaching character limit, but 781 remarks scrolls. I was testing on an iPad Mini 6th Generation

i see it when i tap enter, do we disable multi line? or enter? I did not change the previous behavior of the field only add a character limit

File: PBLogging/Views/SingleSortie/MissionDataView.swift [hunk 1]
```diff
@@ -38,9 +38,11 @@ struct MissionDataView: View {
 
                     SortieTypeView(sortieType: sortie.sortieType)
                     FuelAirlandAirdropView(sortie: sortie)
-                    
-                    TextEditorView(title: "Sortie Comments", text: $sortie.comments)
-                        .padding()
+
+                    TextEditorView(title: "Sortie Comments", text: $sortie.comments.onChange { $0.enforceCharacterLimit(1000) })
+                        .frame(height: 200)
+                        .padding([.horizontal,
```
Comment (tsanders88) [inline PBLogging/Views/SingleSortie/MissionDataView.swift:44]: isn't this just the same as saying .padding()

**Comment by renicklopez** [comment]
> doesn't display character limit

fixed

**Review by thardie2 — APPROVED** [review]

**Pull request merged**